------------------------------------------------------------------------
r572103 | mueller | 2006-08-11 17:58:09 +0200 (Fri, 11 Aug 2006) | 3 lines
Changed paths:
   M /branches/KDE/3.5/kdebase/ksystraycmd/main.cpp

fix argument getting de-quoted
BUG:124116

------------------------------------------------------------------------
Index: ksystraycmd/main.cpp
===================================================================
--- ksystraycmd/main.cpp	(revision 572102)
+++ ksystraycmd/main.cpp	(revision 572103)
@@ -5,6 +5,7 @@
 #include <kcmdlineargs.h>
 #include <kdebug.h>
 #include <klocale.h>
+#include <kprocess.h>
 
 #include "ksystraycmd.h"
 
@@ -94,7 +95,7 @@ int main( int argc, char *argv[] )
   // Read the command
   QString command;
   for ( int i = 0; i < args->count(); i++ )
-    command += QCString( args->arg(i) ) + " ";
+    command += KProcess::quote(QString::fromLocal8Bit( args->arg(i) )) + " ";
   if ( !command.isEmpty() )
       cmd.setCommand( command );
 
